lib/oli/resources/revision.ex

```diff
@@ -37,6 +37,7 @@ defmodule Oli.Resources.Revision do
     field :title, :string
     field :slug, :string
     field :deleted, :boolean, default: false
+    field :ids_added, :boolean, default: false
```

Comment:
NIT: Adding a comment on the meaning of this field might be helpful in the future. In the context of this PR we understand its purpose but probably in some months, it won't be easy to understand why and where we need this field.